docs/guides/software-lifecycle/security/dependency-vulnerability-scanning/README.mdx

```diff
@@ -0,0 +1,127 @@
+- Automating vulnerability detection in repositories
+
+---
+
+## Prerequisites
+**Software:**
+- OCI compliant containers (e.g. Docker, Podman) or other package-manager software dependencies
+- `pre-commit` framework
+
+**Skills:**
+- Basic knowledge of Git hooks and Docker commands
+- Understanding of YAML for pre-commit configuration
+
+---
+
+## Quick Start
+
+**Run a local scan of your container's repository (folder containing the Dockerfile) using [Grype](https://github.com/anchore/grype)**
+
+```bash
+grype dir:.
+```
+
+**⬇️ [.pre-commit-config.yml](.pre-commit-config.yml)**
+
+Download the file above to access the pre-commit configuration file, which includes an example hook for Grype vulnerability scanning. This file should be placed within your local Git repository after installing the pre-commit framework.
+
+---
+
+## Step-by-Step Guide
+
+### Step 1: Setup Automated Local Scanning of Container Vulnerabilities
+
+1. Ensure Grype is installed on your system. You can install Grype from [the official repository](https://github.com/anchore/grype).
+
+    ```bash
+    grype version
+    ```
+
+2. Perform a scan of the local repository for vulnerabilities:
+
+    ```bash
+    grype dir:.
+    ```
+
+3. If you find vulnerabilities, fix them via your package manager. 
+
+### Step 2: Setup Automated Local Scanning of Container Vulnerabilities
+
+⚠️ NOTE: We recommend installing this pre-commit hook only if you have already scanned your repository and addressed any vulnerabilities.
+
+The below steps, once enacted, will ensure that any local `git commit` actions taken will be followed by an automated vulnerability scan. If vulnerabilities at the CRITICAL level are found, the commit will be blocked by default.
+
+1. Install the pre-commit framework via Python:
+   ```bash
+   pip install pre-commit
+   ```
+2. Create a `.pre-commit-config.yaml` file in the root directory of your Git repository with the following content for Grype scanning:
+   <CodeBlock language="yaml">{PreCommitConfigSource}</CodeBlock>
+
+3. Initialize pre-commit in your repository with the new configuration:
+   ```bash
+   pre-commit install
+   ```
+
+4. Grype-based vulnerability scanning should run every time a `git commit` is invoked. The commit will be blocked if CRITICAL level vulnerabilities are found and will ask the developer to fix them prior to committing. 
+
+### Step 3: Set Up Automated Repository Scanning
+- **GitHub**:
+   - We recommend installing the official Grype GitHub action to set up automated dependency vulnerability scanning. Available [at this link](https://github.com/marketplace/actions/anchore-container-scan).
+   - We also recommend setting up GitHub's official Dependabot action to also look for vulnerabilities. See our [GitHub Security Guide](/docs/guides/software-lifecycle/security/github-security) on this. 
+
```

Comment:
The "GitHub" bullet item is an widow here, which suggests that it can just be replaced with text; i.e., instead of:

-   GitHub
    -  We recommend …
    - We also recommend…

Simply say:

For GitHub users, we recommend:

- Installing the …
- Setting up …

When you add GitLab or other hosting platforms, then the two-level bullets are appropriate:

-   GitHub
    -  We recommend …
    - We also recommend…
-   GitLab
    -  We recommend …
    - We also recommend…
-   BitBucket
    -  We recommend …
    - We also recommend…

Reply:
Good point - I incorporated your feedback @nutjob4life. Appreciate the better formatting suggestions.